core-plugins/src/main/java/io/cdap/plugin/batch/connector/FileConnector.java

```diff
@@ -50,7 +50,7 @@
  */
 @Plugin(type = Connector.PLUGIN_TYPE)
 @Name(FileConnector.NAME)
-@Description("File connector for local file system")
+@Description("Connection to browse and sample data from the local file system.")
```

Comment:
Connection -> Connector

Reply:
this is per request of our document writer : https://cdap.atlassian.net/browse/CDAP-18109